pkg/scheduler/apis/config/testing/compatibility_test.go

```diff
@@ -1412,7 +1409,7 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 	}
 }
 
-func TestAlgorithmProviderCompatibility(t *testing.T) {
+func TestComponentConfigCompatibility(t *testing.T) {
```

Comment:
this will be removed in a follow up PR, the test will be part of the defaulting unit tests.